update to the README: description, images and roadmap (#120)
cncf-tags/green-reviews-tooling: README.md

The CNCF Green Reviews Working Group (WG) is an open-source, community-led project that is part of the [CNCF Environmental Sustainability Technical Advisory Group (TAG ENV)](https://github.com/cncf/tag-env-sustainability).

The aim of the Green Reviews WG is to set up infrastructure to measure the sustainability footprint of [CNCF Projects](https://www.cncf.io/projects). 

Measuring the sustainability footprint of software is not an easy task. Our vision is that the WG will compute the sustainability data for every release of a CNCF project that requests a sustainability footprint assessment. To achieve such a vision, our goal is to develop a workflow that can integrate well with the existing software lifecycle of other CNCF projects.

A good way to practically understand the first version of the workflow that the WG is designing is to take a look at the following architecture diagram:

![green reviews workflow](./docs/images/workflow-vision.png)

The WG’s workflow vision is that every release requesting a sustainability footprint assessment will trigger a Github Action specified in the Green Reviews repo that will start a benchmarking pipeline. The pipeline’s job is to:

> 1. Spin up the Equinix Metal resources
> 2. Install Kubernetes and all the needed observability tools

Reviewer: ```suggestion
```

 I think we should remove these 2 points. 

As the cluster is provisioned with opentofu but its pipeline is separate from the measurement pipeline. So it could be confusing.

Reviewer: @AntonioDiTuri Lets keep these 2 points. You convinced me in the meeting! 😄 

Reducing the footprint of the cluster is important to me and I think the WG more widely. I'm hoping we can work on it after the current pipeline work and we have #67 in the backlog.

3. Install the software that will be assessed

> 4. Execute the necessary test cases

Reviewer: ```suggestion
4. Execute the necessary benchmark test cases
```

5. Gather sustainability-related metrics
6. Publish sustainability metrics

> If you are curious and want to discover more a good:

Reviewer 2: ```suggestion
If you are curious and want to discover more, here are some good resources:
```

| Timeline | Goals |
|---|---|
| Q4 '23 / Before & After KubeCon NA  | Announce Green Reviews WG during KubeCon NA Keynotes  |

> |  | Draft of the implementation for the sustainability footprint measurement workflow (tracking issue) |

Reviewer 2: ```suggestion
|  | Draft of the implementation for the sustainability footprint measurement workflow |
```

> |  | Cluster up and running on Equinix (tracking issue) |

Reviewer 2: ```suggestion
|  | Cluster up and running on Equinix |
```
